python_practice/strings_part_one.py

```diff
@@ -27,3 +31,4 @@ def substring_test(substring, string):
         substring = 'foo'
         string = 'foobar'
     """
+    return str.casefold(substring) in str.casefold(string)
```

Comment:
Cool `casefold()` is a method I am not familiar with.  Maybe you could explain it to me.